lib/rambla.ex

```diff
@@ -78,11 +78,30 @@ defmodule Rambla do
                 reduce: %{},
                 do: (acc -> Map.update(acc, name, [service], &[service | &1]))
 
+  @services :rambla |> Application.get_all_env() |> Keyword.get(:services, [])
```

Comment:
Indeed.

Still, my concern is we should not ruin `mock` and `stub` in tests by runtime settings. The config in runtime is surely something we need to introduce, but it should be done carefully. The typical config has no services but `mock`/`stub` in `:test` and we should keep it that way.

Let me think about it for a while.

Reply:
Cool.

What we would also need to mention this in docs. Please also make sure that configs are _merged_ so that one would not need to fully copy-paste configs from `config.exs` into `runtime.exs`.

Reply:
@anthony-gonzalez-kantox please confirm configs are merged and I’ll merge&publish.